src/Homie/Boot/BootConfig.cpp

```diff
@@ -175,20 +176,21 @@ void BootConfig::_onWifiStatusRequest(AsyncWebServerRequest *request) {
 
   json["status"] = status;
   String output;
-  json.printTo(output);
+  serializeJson(json, output);
```

Comment:
`serializeJson(generatedJsonDocument, output);` can be used here when the issue above is resolved.